tests/plugins/crop_generators/test_mixup_crop_generator.py

```diff
@@ -56,13 +58,28 @@ def metadata(config, make_struct):
     meta.splitMetadata(train_rule, val_rule)
     return meta
 
+@pytest.fixture(scope="function")
```

Comment:
This piece of code seems to be reused over and over. Can we generalize it somehow?

Reply:
Probably we could make shared fixtures, as ```locations``` is not the only one fixture we use over and over. https://stackoverflow.com/questions/46005796/reuse-pytest-fixtures